openghg/dataobjects/_searchresults.py

```diff
@@ -249,3 +251,41 @@ def _retrieve_dataset(
             return recombine_datasets(
                 keys=keys, sort=sort, elevate_inlet=elevate_inlet, attrs_to_check=attrs_to_check
             )
+
+    @staticmethod
+    def df_to_table_console_output(df: DataFrame) -> None:
+        """
+        Process the DataFrame and display it as a formatted table in the console.
+
+        Args:
+            df (DataFrame): The DataFrame to be processed and displayed.
+
+        Returns:
+            None
+        """
+        from rich import print
```

Comment:
Another small update I just considered is that we should probably catch the ImportError for the rich package if this is not present and include a bypass to use the old style. e.g. something like:

```suggestion
        try:
            from rich import print
            from rich.table import Table, box
        except ImportError:
            # Should likely add a logger line here but would also need to set up the logging
            print("Unable to use rich package to display search results. Please install rich to do so.")
            return None
```

Reply:
> Another small update I just considered is that we should probably catch the ModuleNotFoundError for the rich package if this is not present and include a bypass to use the old style. e.g. something like:
> 
> ```
>         try:
>             from rich import print
>             from rich.table import Table, box
>         except ModuleNotFoundError:
>             # Should likely add a logger line here but would also need to set up the logging
>             print("Unable to use rich package to display search results. Please install rich to do so.")
>             return None
> ```

Won't the new release means re-running the requirements.txt?